app/helpers/description_change_validation_request_helper.rb

```diff
@@ -8,4 +8,9 @@ def change_rejected?(description_change_validation_request)
   def sequence_description(description_change_validation_request)
     "description##{description_change_validation_request.sequence}"
   end
+
+  def format_datetime(datetime_str)
+    datetime = DateTime.parse(datetime_str)
+    datetime.strftime("%-d %B %Y")
```

Comment:
It might be best to use `to_fs` here, as that makes it easier to keep the formats consistent (it uses the ones defined in `config/initializers/date_formatter.rb`).

```suggestion
    datetime.to_date.to_fs
```

Reply:
I hadn't spotted that method... makes life so much easier!